cloud/pkg/edgecontroller/manager/configmap.go

```diff
@@ -1,7 +1,7 @@
 package manager
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
No usage referring to this change?

Reply:
Yeah, it is not a necessary modification. It has been restored